scripts/make-g-cloud-live.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
does this line tell it to run in python3?

Reply:
If you call it directly with `./<script name>`, it'll make it look for a Python3 interpreter, yeah. You can still force it to run in Python 2 though. This is called a shebang and is interpreted by bash when executing the file.

Reply:
Ahh, awesome, that's a good explanation, thank you :)

Reply:
I slightly lied. It's not interpreted by bash, it's interpreted by the 'program loader'. I don't know what that is. But more details and links off from here: https://en.wikipedia.org/wiki/Shebang_(Unix) :)